Detections/AzureFirewall/SeveralDenyActionsRegistered.yaml

```diff
@@ -0,0 +1,34 @@
+id: 
+name: Several deny actions registered
+description: |
+  'This Creates an incident when a source has more than 1 registered deny action in Azure Firewall.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AzureFirewall
+    dataTypes: 
+      - AzureDiagnostics
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 1
+tactics:
+  - InitialAccess
```

Comment:
I think this is more like CommandAndControl.  This is a deny on an outbound connection, correct?  To me, this would indicate the system has executed a payload and attempting to reach back to a malicious website.  I am open to a discussion, just want to make sure we are hitting the mark on the Tactic.

Reply:
I agree that the tactics isn't really InitialAccess. Edited to CommandAndControl (for outbound connections), Discovery (for discovering resources) and LateralMovement (for moving between resources)